requirements.txt

```diff
@@ -1,9 +1,9 @@
 boto3
-azure-mgmt-compute==30.1.0
+azure-mgmt-compute==30.4.0
 azure-mgmt-storage==21.1.0
-azure-mgmt-resource==23.0.1
-azure-storage-blob==12.17.0
-azure-identity==1.14.0
+azure-mgmt-resource==23.1.0b2
```

Comment:
Are you sure you want to include a beta version?

Reply:
I gave you extend answer in PM . Here I will give short answer - it is choice of MS developers